ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -1318,6 +1318,10 @@ private KsqlConfig(final ConfigGeneration generation,
     this.ksqlStreamConfigProps = ksqlStreamConfigProps;
   }
 
+  public boolean keyExistsInConfigMap(final String key) {
```

Comment:
what's this for? it seems like it's never used outside your unit test.

Reply:
Good catch. I was using this for my validation before we agreed on the allow list.